libr/anal/meta.c

```diff
@@ -727,6 +727,34 @@ static int meta_print_item(void *user, const char *k, const char *v) {
 	return 1;
 }
 
+R_API void r_meta_list_offset(RAnal *a, ut64 addr, char input) {
+	RAnalMetaUserItem ui = { a, 0, 0, NULL, NULL, 0, NULL };
+	SdbList *ls = sdb_foreach_list (DB, true);
+	SdbListIter *lsi;
+	SdbKv *kv;
+
+	ls_foreach (ls, lsi, kv) {
```

Comment:
this is  very slow. see how the meta is accessed in other functions here, to reduce the scope of iterations it shifts the offset and uses that as a key to resolve the list of values that match a range of offsets.

Reply:
just do r-meta-find

Reply:
uhm. chk this:
```

R_API RAnalMetaItem *r_meta_find_in(RAnal *a, ut64 at, int type, int where) {
        char *res = meta_inrange_get (a, at, 1);
```